packages/stylelint-config/rules/stylelint-config-lmc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  rules: {
+    /**
+     * Rules added from `old stylelint list` not included in any extension package:
+     */
+    'declaration-no-important': [
+      true,
+      {
+        severity: 'warning',
+      },
+    ],
+    'selector-max-class': 5,
+    'selector-max-id': 1,
+    'font-weight-notation': 'numeric',
+    'unit-allowed-list': ['em', 'rem', '%', 's', 'px', 'pt', 'ex', 'deg', 'cm', 'fr', 'vh', 'vw'],
+    'color-named': 'never',
+
+    /**
+     * To discuss - rule modifications due to extension packages implementation - wasn't a problem before, now it is:
+     */
+    // Variables syntax pattern should be unified, not mixed like kebab-case and camel-case. Use kebab-case only.
+    'scss/dollar-variable-pattern': null,
```

Comment:
Could you please add links to the documentation, for this line it is https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-pattern/README.md

Reply:
Documentation links were added. New comments on rules to be enabled was added as well and waiting for feedback.